rtb/norme_di_progetto_ver0.5.0.typ

```diff
@@ -531,9 +540,182 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+Ha il compito di:
+- Implementare la Specifica Tecnica scritta dal Progettista;
+- Scrivere codice mantenibile, che rispetti le `Norme di Progetto`;
+- Creare test per la verifica e validazione del codice;
+- Redigere il manuale utente.
+
+==== Ticketing
+Il gruppo adotta l'*Issue Tracking System* (ITS) interno di GitHub. GitHub permette una gestione semplice e chiara dei compiti da svolgere tramite la project board e le issue: le issues vengono create molto velocemente e possono essere chiuse automaticamente a lavoro svolto.\
+È compito del responsabile creare le issues ed assegnarle ai vari membri del gruppo, il cui stato di avanzamento è consultabile all'interno della board.\
+Le Issue sono create dal Responsabile e sono composte da:
+- *Titolo*: identifica in modo univoco il compito da svolgere;
+- *Descrizione*: descrizione chiara dell'attività da svolgere e riferimento al verbale in cui è stata presa tale decisione per avere più informazioni riguardo ad essa;
+- *Assegnatario*: il componente del gruppo incaricato a svolgere l'attività;
+- *Verificatore*: il componente incaricato a garantire che l'attività venga svolta senza errori e rispettando gli standard;
+- *Milestone*: la release in cui deve essere pubblicato il risultato dell'attività;
+- *Etichetta*: la categoria a cui appartiene quella attività;
+- *Stato*: avanzamento del task.
+Ogni qualvolta ci sia la necessità di portare a termine un compito è necessario seguire la seguente procedura:
++ Il responsabile, dopo aver concordato in una riunione il da farsi, crea una nuova Issue con stato "to do" su GitHub e la assegna;
++ All'inizio del lavoro di produzione, l'assegnatario cambia la issue  stato, passando da "to do" ad "in progress", inoltre crea un nuovo branch apposito staccato dal ramo principale per eseguire l'attività;
++ Contestualmente l'assegnatario apre una pull request su GitHub in stato draft, per rendere pubblico il suo lavoro ancora prima che sia pronto. Nella descrizione della pull request è importante che scriva la parola chiave `closes #X` (dove X è l'id della issue) per chiudere la issue una volta che la pull request sarà approvata e mergiata nel ramo principale;
++ Finito il lavoro di produzione, viene modificata la pull request dallo stato draft allo stato ready for review e viene assegnato il Verificatore;
++ Il Verificatore si controlla il lavoro svolto e:
+  - Se la verifica ha esito *positivo*:
+    + Il Verificatore approva su GitHub la pull request e chiede al responsabile di mergiarla nel ramo principale ;
+    + La Issue viene marcata "Done" su GitHub automaticamente.
+  - Se la verifica ha esito *negativo*:
+    + Il Verificatore rilascia una lista di cambiamenti suggeriti nella pull request;
+    + L'incaricato apporta le modifiche suggerite e si torna al punto 4.
+
```

Comment:
Occhio ai termini inglesi, provo a elencare quelli trovati in questo paragrafo, sono presenti più volte quindi cerca di mettere il corsivo in tutte le occorrenze:
- project board
- issue (dove non lo è la metterei in minuscolo, dimmi se non ho capito perché è in maiuscolo)
- issues
- board
- release
- task
- "to do" va bene tra virgolette ma comunque sarebbe da aggiungere il corsivo
- "in progress" come "to do"
- pull request
- draft
- 'closes #X' metterei tra virgolette e non in monospace
- mergiata è italianizzato, cambierei con "ed effettuato il _merge_"
- ready for review
- mergiarla è italianizzato, cambierei con "effettuare il _merge_"
- "Done"

A parte, "Verificatore" in "...viene assegnato il Verificatore" lo metterei in minuscolo